lessons/pydata/clt/index.ipynb

```diff
@@ -0,0 +1,690 @@
+{
```

Comment:
<p>&gt; <em>Přibližně</em>&nbsp;proto, že získané normální rozložení není zcela ideálního tvaru.</p><p> </p><p><em>Přibližně</em>&nbsp;proto, že získané rozložení nemá ideální "normální" tvar. A při výběru konečného množství vzorků (navíc z konečné populace) ani mít nemůže.</p><p> </p><p>&gt; Jaká je tedy ta správná velikost vzorku?</p><ul><li>Tohle mi přijde hodně postavené na víře (a pominu teď, že jsem slyšel víceméně stejná čísla, je samotné nerozporuju). Nechceš to podložit, když ne vysvětlením proč (pro to je tu asi málo místa), tak odkazem na nějaký relevantní zdroj?</li><li>Je to univerzální na výběr vzorku z populace, nebo jsou některé z těch kritérií specifické pro výzkumy veřejného mínění?</li></ul><p> </p><p>&gt; Existuje několik formulí,&nbsp;</p><p> </p><p>Existuje několik vzorců,</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>